keep/rulesengine/rulesengine.py

```diff
@@ -132,40 +132,65 @@ def run_rules(self, events: list[AlertDto]):
 
             group_status = self._calc_group_status(group.alerts)
             # get the payload of the group
-            # todo: this is not scaling, needs to find another solution
-            # group_payload = self._generate_group_payload(group.alerts)
-            # create the alert
-            event = {
+            # # todo: this is not scaling, needs to find another solution
+            # # group_payload = self._generate_group_payload(group.alerts)
+            # # create the alert
+            # event = {
+            #     "name": group_name,
+            #     "id": group_fingerprint,
+            #     "description": group_description,
+            #     "lastReceived": group_attributes.get("last_update_time"),
+            #     "severity": group_severity,
+            #     "source": group_source,
+            #     "status": group_status,
+            #     "pushed": True,
+            #     "group": True,
+            #     # "groupPayload": group_payload,
+            #     "fingerprint": group_fingerprint,
+            #     **group_attributes,
+            # }
+            # group_alert = create_alert_db(
+            #     tenant_id=self.tenant_id,
+            #     provider_type="group",
+            #     provider_id=rule.id,
+            #     # todo: event should support list?
+            #     event=event,
+            #     fingerprint=group_fingerprint,
+            # )
+            # elastic_client = ElasticClient(self.tenant_id)
+            # elastic_client.index_alert(
+            #     alert=AlertDto(event_id=str(group_alert.id), **event),
+            # )
+            # grouped_alerts.append(group_alert)
+            # self.logger.info(f"Created alert {group_alert.id} for group {group.id}")
+            
+            # Create the incident
+            incident_data = {
```

Comment:
Not sure whether this is a proper way to add additional information for an incident, let me know what should be included.